ably/proto/message.go

```diff
@@ -44,18 +260,22 @@ func (m *Message) MemberKey() string {
 // was encrypted.
 func (m *Message) DecodeData(opts *ChannelOptions) error {
```

Comment:
It still feels a bit strange that `EncodeData` and `DecodeData` are exported methods, since I really wouldn't expect users of the package to ever have to call them

Reply:
What happens if you call `DecodeData` twice on the same `Message`?